common/versions.yml

```diff
@@ -0,0 +1,6 @@
+# This file is generated by https://github.com/gpii-ops/gpii-version-updater.
```

Comment:
Keep an eye on this file and the old location when you merge. If version.yml changes, I'm not 100% certain git will do the right thing.